go.mod

```diff
@@ -48,6 +48,7 @@ require (
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/segmentio/backo-go v1.0.1 // indirect
 	github.com/sirupsen/logrus v1.9.3 // indirect
+	github.com/stretchr/objx v0.5.1 // indirect
```

Comment:
Why these changes?

Reply:
this might be imported by the `"github.com/stretchr/testify/mock"` package used in the testcase